pom.xml

```diff
@@ -75,15 +75,23 @@
                 <version>1.0.20</version>
                 <executions>
                     <execution>
-                        <goals>
-                            <goal>backport7to6</goal>
-                        </goals>
-                        <configuration>
-                            <classified>true</classified>
-                        </configuration>
+                        <!--<goals>-->
+                            <!--<goal>backport7to6</goal>-->
+                        <!--</goals>-->
+                        <!--<configuration>-->
+                            <!--<classified>true</classified>-->
+                        <!--</configuration>-->
                     </execution>
```

Comment:
I realize that, what I didn't know was that this PR is not intended for merging (as you said elsewhere). Thus the "draft" aspect of it.